django/home/models.py

```diff
@@ -940,6 +940,32 @@ def __str__(self):
         return "submission {} by {}".format(self.title, self.submitter)
 
 
+class DigestArchive(index.Indexed, models.Model):
+    """
+    represents a single issue of the quarterly digest with a pdf archive in the static directory
+    """
+
+    class Seasons(models.IntegerChoices):
+        SPRING = 1, _("Spring")
+        SUMMER = 2, _("Summer")
+        FALL = 3, _("Fall")
+        WINTER = 4, _("Winter")
+
+    year_published = models.IntegerField()
```

Comment:
should we also keep track of a `date_created` or something equivalent for internal book-keeping? Or we could have a single `publication_date` and extract the year from it

Reply:
to clarify, something like `publication_date = models.DateField()`

not sure if it's necessary to keep `last_modified` + `date_created` auto fields here like we do for other comses models where edit date provenance is important, thoughts?

Reply:
Originally there was a `publication_date` field but it seemed to me the publication date may be ambiguous. On second thought, I suppose the end date of the given period may be it, or close enough? 

![Screenshot from 2023-03-23 09-44-23](https://user-images.githubusercontent.com/46429375/227275320-cf7d7bc4-0c1a-44c2-b3f4-76bfffe7c87a.png)

Reply:
Yep, the end date of the period the digest covers. Now it's always 12/16-3/15, 3/16-6/15, 6/16-9/15, 9/16-12/15 though in the early days we'd make the interval non-standard to cover whenever we actually got around to sending the thing out